ark/utils/load_utils.py

```diff
@@ -147,6 +152,15 @@ def load_imgs_from_tree(data_dir, img_sub_folder=None, fovs=None, channels=None,
         # get the corresponding indices found in channels_no_delim
         channels_indices = [channels_no_delim.index(chan.split('.')[0]) for chan in all_channels]
 
+        # verify if channels from user input are present in `all_channels`
+        all_channels_no_delim = [channel.split('.')[0] for channel in all_channels]
+        try:
+            misc_utils.verify_same_elements(all_channels_in_folder=all_channels_no_delim,
+                                            all_channels_detected=channels_no_delim)
+        except ValueError as _:
```

Comment:
I don't think we need to catch the error, just specifying the names like you did above is good

Reply:
So just changing the variable names in verify_same_elements is alright?

Reply:
yup exaclty